charts/lfx-v2-project-service/values.yaml

```diff
@@ -19,10 +19,24 @@ initImage:
   # pullPolicy is the init container image pull policy
   pullPolicy: IfNotPresent
 
-# ingress is the configuration for the ingress routing
-ingress:
-  # hostname is the hostname of the ingress
-  hostname: lfx-api.k8s.orb.local
+# traefik is the configuration for Traefik Gateway API routing
+traefik:
+  # gateway specifies the platform Gateway to attach to
+  gateway:
+    name: lfx-platform-gateway
+    namespace: lfx
+
+# lfx is the configuration for LFX platform
+lfx:
+  # domain is the base domain for routing
+  domain: k8s.orb.local
+  # namespace is the target namespace for deployment
+  namespace: lfx
+
+# service is the configuration for the Kubernetes service
+service:
+  # port is the service port
+  port: 8080
```

Comment:
If this is now configurable, can you also update the service and deployment to use the port from values?